Client/Ecosia/Analytics/Analytics.swift

```diff
@@ -29,134 +29,46 @@ final class Analytics {
         }
     }
     
-    func open(_ action: Action.Activity) {
+    func activity(_ action: Action.Activity) {
         tracker.track(SPStructured.build {
             $0.setCategory(Category.activity.rawValue)
             $0.setAction(action.rawValue)
             $0.setLabel("inapp")
         })
     }
-    
-    func screen(_ label: Label.Screen) {
-        tracker.track(SPStructured.build {
-            $0.setCategory(Category.navigation.rawValue)
-            $0.setAction(Action.view.rawValue)
-            $0.setLabel(label.rawValue)
-        })
-    }
-    
-    func newTab(_ origin: Property) {
-        tracker.track(SPStructured.build {
-            $0.setCategory(Category.browser.rawValue)
-            $0.setAction(Action.open.rawValue)
-            $0.setLabel("new_tab")
-            $0.setProperty(origin.rawValue)
-        })
-    }
-    
-    func features(_ label: Label.Features, origin: Property) {
+
+    func browser(_ action: Action.Browser, label: Label.Browser, property: Property? = nil) {
```

Comment:
Property is not being used, please removed it from the parameters

Reply:
great catch! it should be used though, so I'll add it to the code 👍